src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -46,7 +46,6 @@ private Case createSkeletonCase(CollectionCase collectionCase) {
     Case skeletonCase = new Case();
     skeletonCase.setSkeleton(true);
     skeletonCase.setCaseId(UUID.fromString(collectionCase.getId()));
-    skeletonCase.setCollectionExerciseId(collectionCase.getCollectionExerciseId());
```

Comment:
the collex ID is the way of knowing which cases belong together (i.e. census vs. CCS) and it goes to RH

Reply:
Now set in yml and always put on skeleton case